contracts/DankFaucet.sol

```diff
@@ -104,4 +108,20 @@ contract DankFaucet is usingOraclize {
    * Simple function to allow for adding ETH to the contract.
    */
   function sendEth() public payable { }
+
+  /**
+   * Allow the owner to reset the DANK we've allocated to a phone number.
+   */
+  function resetAllocatedDankForPhoneNumber(bytes32 hashedPhoneNumber, uint dank) auth {
```

Comment:
What is the purpose of the dank parameter here if we can only assign allotment amounts to users? Isn't it better to remove that parameter and just remove the phone entry in the map?

Reply:
I went back and forth on this one. It was pretty easy to add in the amount that way we had more control with just a little bit of flexibility. That flexibility would only be useful in the case where we change the initial allotment and changed the logic to allow folks to "top off" once that allotment had changed. It doesn't make much sense right now so I can remove that argument and just remove the entry from the map.